float32 x
float32 y
float32 heading
bool mode_heading
